include/fmt/std.h

```diff
@@ -28,6 +31,16 @@
 #  endif
 #endif
 
+#if FMT_HAS_INCLUDE(<cxxabi.h>) || defined(__GLIBCXX__)
+#  include <cxxabi.h>
+#  ifndef __GABIXX_CXXABI_H__
```

Comment:
What is `__GABIXX_CXXABI_H__` check for?

Reply:
For Android.
See comment from Boost:

> // For some archtectures (mips, mips64, x86, x86_64) cxxabi.h in Android NDK is implemented by gabi++ library
> // (https://android.googlesource.com/platform/ndk/+/master/sources/cxx-stl/gabi++/), which does not implement
> // abi::__cxa_demangle(). We detect this implementation by checking the include guard here.

https://github.com/boostorg/core/blob/162a4e1d24d1cab4f24cbca9489203d891f352e6/include/boost/core/demangle.hpp#L33-L35

Reply:
Could you add a comment here too?

Reply:
Done.
And I renamed ``FMT_HAS_CXXABI_H`` to ``FMT_HAS_ABI_CXA_DEMANGLE``.